docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3'
+services:
+  redis:
+    image: redis
+    ports:
+      - 6379:6379
+
+  aelf-node:
+    depends_on:
+      - redis
+    working_dir: /app
+    image: aelf/node
```

Comment:
Don't call it this name. Maybe use `aelf/standalone-testing-node`

Reply:
Modified.